cloud/aws/modules/setup/secrets.tf

```diff
@@ -0,0 +1,90 @@
+# Firstly create a random generated password to use in secrets.
+resource "random_password" "postgres_password" {
+  length           = 16
+  special          = true
+  override_special = "_%@"
+}
+
+# Creating a AWS secret for postgres_password
+resource "aws_secretsmanager_secret" "postgres_password_secret" {
```

Comment:
we probably want to create a separate KMS for all these secrets, and not use the default one. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/secretsmanager_secret#kms_key_id